### ISSUE TITLE
SNES: different English translation versions of Super Famicom Wars

### ISSUE THREAD

**Issue by tjanas** [SmokeMonsterPacks/EverDrive-Packs-Lists-Database#122]
Both version 1.0 and 1.1 exist. 1.1 is the [latest version](https://www.romhacking.net/translations/3354/), there is no reason to include version 1.0.

```
84e9f52266d36dee9fab12211ded5d0ad6db59d98b652a5eb1e5bd9933aef0c3	SD2SNES/3 Nintendo Power Cartridge Service/Super Famicom Wars English (J) (NP) T+Eng v1.1 Optiroc.sfc	e8309511f7bb903059de7e166c2d08cd971792e5	96501e0e8d2f7435ac6ac5727571f8bb	45cbdf8d

4a8a2545442485eba7f3eafa9e3eae4f64f893971666f873a4236c38bc53862f	SD2SNES/4 Game Series Collections/Genre/Strategy/1 All Strategy Games/Super Famicom Wars (J) T+Eng v1 Optiroc.sfc	1e3f40712b89c83a98fe4ece02ee53c25cdd0463	94de7a17a1a04083f5e6d52034432af3	41850b51

4a8a2545442485eba7f3eafa9e3eae4f64f893971666f873a4236c38bc53862f	SD2SNES/4 Game Series Collections/Genre/Strategy/2 Sort By Game Type/Strategy - Turn-Based/Super Famicom Wars (J) T+Eng v1 Optiroc.sfc	1e3f40712b89c83a98fe4ece02ee53c25cdd0463	94de7a17a1a04083f5e6d52034432af3	41850b51

4a8a2545442485eba7f3eafa9e3eae4f64f893971666f873a4236c38bc53862f	SD2SNES/4 Game Series Collections/Genre/Strategy/3 Japan & Europe Exclusives/Strategy - Japan Exclusives (English Friendly)/Super Famicom Wars (J) T+Eng v1 Optiroc.sfc	1e3f40712b89c83a98fe4ece02ee53c25cdd0463	94de7a17a1a04083f5e6d52034432af3	41850b51

84e9f52266d36dee9fab12211ded5d0ad6db59d98b652a5eb1e5bd9933aef0c3	SD2SNES/4 Translations/English/English N-Z/Super Famicom Wars English (J) (NP) T+Eng v1.1 Optiroc.sfc	e8309511f7bb903059de7e166c2d08cd971792e5	96501e0e8d2f7435ac6ac5727571f8bb	45cbdf8d

84e9f52266d36dee9fab12211ded5d0ad6db59d98b652a5eb1e5bd9933aef0c3	SD2SNES/4 Translations/Selections/Super Famicom Wars English (J) (NP) T+Eng v1.1 Optiroc.sfc	e8309511f7bb903059de7e166c2d08cd971792e5	96501e0e8d2f7435ac6ac5727571f8bb	45cbdf8d
```

**Comment by frederic-mahe** [comment]
You are right, we don't need different revisions of the same translation patch.

For future reference, I think it is ok to preserve different revisions of licensed games and homebrew official releases. The first application that comes to mind is speed running, where runners play on specific game revisions (often v1.0).

**Comment by frederic-mahe** [comment]
fixed in e6e143f4ac454b2b47855492412591399475b1c8
